src/components/tx-flow/flows/TokenTransfer/ReviewSpendingLimitTx.tsx

```diff
@@ -96,9 +99,14 @@ const ReviewSpendingLimitTx = ({
       setTxFlow(undefined)
     } catch (_err) {
       const err = asError(_err)
-      logError(Errors._801, err)
-      setIsSubmittable(true)
-      setSubmitError(err)
+      if ('code' in err && err.code === ErrorCode.ACTION_REJECTED) {
```

Comment:
Might be good to create a utility for this, perhaps in `src/utils/wallets.ts`.

Actually, I've just checked and that file already has sWalletRejection :)

Reply:
Perfect, thanks!